src/Kernel32.Tests/Kernel32Facts.cs

```diff
@@ -39,4 +40,21 @@ public unsafe void GetStartupInfo_Title()
         Assert.NotNull(startupInfo.Title);
         Assert.NotEqual(0, startupInfo.Title.Length);
     }
+
+    [Fact]
+    public void GetHandleInformation_DoesNotThrow()
+    {
+        var manualResetEvent = new ManualResetEvent(false);
+        GetHandleInformation(manualResetEvent.SafeWaitHandle, out var lpdwFlags);
```

Comment:
Add `Assert.True` around the call so your test will fail if Win32 rejects your input args.